packaging/sunshine.rb

```diff
@@ -53,6 +56,7 @@ def install
     ENV["BRANCH"] = "@GITHUB_BRANCH@"
     ENV["BUILD_VERSION"] = "@BUILD_VERSION@"
     ENV["COMMIT"] = "@GITHUB_COMMIT@"
+    ENV["LIBRARY_PATH"] = "/usr/local/opt/icu4c/lib"
```

Comment:
This location changes depending if it's arm or x86_64.

Reply:
As far as my understanding goes, it does not. This is the path brew install icu4c as a keg-only library, it's not the system installed version. This should use the same architecture as the brew architecture installed.

I did not test this assumption, do you have any resource I could verify?

Reply:
I mean the homebrew install location is different depending on the architecture.

These are the two install paths for homebrew... maybe you can work from that?

```
/opt/homebrew
/opt/local
```

Reply:
Oh, I see, I improved on that, now brew itself tells me the path